src/test/scala/it/bitrock/dvs/api/JsonSupportSpec.scala

```diff
@@ -59,15 +59,15 @@ class JsonSupportSpec extends BaseSpec {
     }
 
     "parse StartTop" in {
-      read(startTop) { result: StartTops.type => result.`@type` shouldBe "startTop" }
+      read(startTop) { result: StartTops => result.`@type` shouldBe "startTop" }
```

Comment:
please also add the second field here, we're testing the deserialisation

Reply:
Added test for rate